lib/akamod/set_prop.py

```diff
@@ -81,7 +81,8 @@ def unset_prop(body, ctype, prop=None, condition=None, condition_prop=None):
         "mwdl_exclude": lambda v: (v[0] == "collections" or
                                    v[0] == "findingAids"),
         "hathi_exclude": lambda v: "Minnesota Digital Library" in v,
-        "finding_aid_title": lambda v: v[0].startswith("Finding Aid")
+        "finding_aid_title": lambda v: v[0].startswith("Finding Aid"),
+        "usc_no_contributor": lambda v: not v[0].get("contributor", False)
```

Comment:
It looked like, in that case, `get_prop` would throw a `KeyError` here: https://github.com/dpla/ingestion/blob/add-usc-no-contributor-check/lib/akamod/set_prop.py#L92 because "contributor" would not exist as a key of the `sourceResource` hash.  I ran a test with `get_prop` in the interpreter and saw that it would do that.